### DISCUSSION TITLE
New to Laragon and having problem nginx instead of Apache

### DISCUSSION THREAD

**Discussion by dinnyt** [leokhoa/laragon#851] [General]
Hi, I don't have nginx enabled in my Laragon settings:

![image](https://github.com/user-attachments/assets/cf21f1f8-f26d-416e-9f43-8dd339201017)

but everytime I try to open my project to view it in the web browser, I get this for both the localhhost and the project.test page.

![image](https://github.com/user-attachments/assets/bc492e9f-45c7-430e-8dff-bf4833a6975f)

How do I solve this please, so that I can see my projects that are supposed to be running on the apache server?

I'm using PHP

my app url is this in the project:

  'url' => env('APP_URL', 'http://localhost'),

and my project config file says that it is using Apache settings:

<VirtualHost *:80> 
    DocumentRoot "${ROOT}"
    ServerName ${SITE}
    ServerAlias *.${SITE}
    <Directory "${ROOT}">
        AllowOverride All
        Require all granted
    </Directory>
</VirtualHost>

<VirtualHost *:443>
    DocumentRoot "${ROOT}"
    ServerName ${SITE}
    ServerAlias *.${SITE}
    <Directory "${ROOT}">
        AllowOverride All
        Require all granted
    </Directory>

    SSLEngine on
    SSLCertificateFile      C:/laragon/etc/ssl/laragon.crt
    SSLCertificateKeyFile   C:/laragon/etc/ssl/laragon.key
 
</VirtualHost>

So how do I access the correct site and not get the nginx welcome page?

Thanks,
DT

**Comment by Pen-y-Fan** [comment]
You have another installation of Nginx running on your computer.

In Laragon menu > Tools > Netstat and looking at port 80, you will see Nginx is using the port to listen for incoming requests from HTTP. Uninstall or stop it if necessary. 

You can then start Laragon. Or in Laragon preferences choose another port e.g. 8080 for Apache instead.

**Comment by dinnyt** [comment]
Hi Pen y Fan,

 

I am using Windows for this project and have a previous Ubuntu on my laptop.  I have found there is a previous version of nginx there.  The download was nginx.8.gz last year.  

 

Could you please provide me with explicit instructions on how I stop it from listening to Laragon?

 

Thank you,

 

Dinny

**Reply by Pen-y-Fan** [reply]
The installation of Nginx is specific to your computer. Google how to uninstall Nginx in Ubuntu and follow the instructions provided there.

**Reply by dinnyt** [reply]
Thank you, changing the port for Apache to 8080 the issue.  I do find it interesting, though, that the nginx port in Laragon was 8080 (not enabled),  the listening port on the Ubuntu nginx is 110, whilst the original listening port on Apache was 80.
